dotCMS/src/main/java/com/dotmarketing/util/StringUtils.java

```diff
@@ -441,4 +441,37 @@ public static boolean hasWhiteSpaces(final String in){
       return pattern.matcher(in).find();
    }
 
+    /**
+     * Method to convert camel case convention into snake case

```

Comment:
Add an example of an input string and the output

Reply:
This feedback has been addressed